WordPress/Classes/Extensions/UINavigationController+Helpers.swift

```diff
@@ -33,4 +33,17 @@ extension UINavigationController {
             }
         }
     }
+
+    /// Fixes a crash in iOS 13 (#12882) where presenting a UIDocumentMenuViewController in a webView
+    /// doesn't rautomattically ecognize the location for presenting the menu hence the crash.
+    /// The warning is probably a bug on iOS or on WebKit since replacing with UIDocumentPickerViewController doesn't prevent the crash.
```

Comment:
This is an iOS 13 bug. I hope this will be fixed by Apple in the near future, then we can remove that override. Do you think that would be useful information to have here?

Reply:
Ok. updated the comments